fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/RecordQueryPlanner.java

```diff
@@ -917,8 +1005,9 @@ private ScoredPlan planRemoveDuplicates(@Nonnull PlanContext planContext, @Nonnu
             if (planContext.commonPrimaryKey == null) {
                 return null;
             }
-            return new ScoredPlan(new RecordQueryUnorderedPrimaryKeyDistinctPlan(plan.plan), plan.unsatisfiedFilters, plan.indexFilters, plan.sargedComparisons, plan.score,
-                    false, plan.flowsAllRequiredFields, plan.includedRankComparisons);
+            return new ScoredPlan(new RecordQueryUnorderedPrimaryKeyDistinctPlan(plan.info),
+                    plan.unsatisfiedFilters, plan.indexFilters, plan.sargedComparisons, plan.score,
+                    false, plan.isStrictlySorted, plan.flowsAllRequiredFields, plan.includedRankComparisons);
```

Comment:
Note to reviewer: I am inheriting `isStrictlySorted` from `plan` which used to be false.